staging/src/github.com/kubeedge/viaduct/pkg/client/quic.go

```diff
@@ -4,8 +4,8 @@ import (
 	"fmt"
 	"sync"
 
-	"github.com/lucas-clemente/quic-go"
-	"k8s.io/klog/v2"
+	quic "github.com/lucas-clemente/quic-go"
+	klog "k8s.io/klog/v2"
```

Comment:
"github.com/lucas-clemente/quic-go"
"k8s.io/klog/v2"
is enough. same with other places, but I don't think it's going to affect anything. : )

Reply:
Fixed. Please review again.